modules/admin/admin-event/src/main/java/com/enonic/xp/admin/event/impl/EventApiHandler.java

```diff
@@ -0,0 +1,75 @@
+package com.enonic.xp.admin.event.impl;
+
+import java.util.List;
+
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+import com.enonic.xp.admin.event.impl.json.EventJsonSerializer;
+import com.enonic.xp.event.Event;
+import com.enonic.xp.event.EventListener;
+import com.enonic.xp.portal.universalapi.UniversalApiHandler;
+import com.enonic.xp.portal.websocket.WebSocketManager;
+import com.enonic.xp.web.HttpStatus;
+import com.enonic.xp.web.WebRequest;
+import com.enonic.xp.web.WebResponse;
+import com.enonic.xp.web.websocket.WebSocketConfig;
+import com.enonic.xp.web.websocket.WebSocketEvent;
+import com.enonic.xp.web.websocket.WebSocketEventType;
+
+@Component(property = {"applicationKey=admin", "apiKey=event", "displayName=Event API", "allowedPrincipals=role:system.admin.login",
+    "mount=true"})
```

Comment:
```suggestion
    "mount=false"})
```

Until this API is stable we don't want to expose it